.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -1105,6 +1108,205 @@ stages:
       condition: succeededOrFailed()
       continueOnError: true
 
+- stage: asan_tests_windows_profiler
```

Comment:
if we don't want to change the name, could we at least leave a comment about what it means? 😄 
```suggestion
# Address sanitizer tests
- stage: asan_tests_windows_profiler
```

Reply:
Make sense.
Done here 2975012b00f74259f02da19e743655d0507f0d0b